.travis.yml

```diff
@@ -16,6 +16,8 @@ env:
     - PUPPET_GEM_VERSION="~> 3.5.1"
     - PUPPET_GEM_VERSION="~> 3.6.0"
     - PUPPET_GEM_VERSION="~> 3.7.0"
```

Comment:
oh! Travis checks are quite outdated. Please add these lines here too:
<pre>
    - PUPPET_GEM_VERSION="~> 3.8.0"
    - PUPPET_GEM_VERSION="~> 3" FUTURE_PARSER="yes"
    - PUPPET_GEM_VERSION="~> 4.0.0"
    - PUPPET_GEM_VERSION="~> 4.1.0"
    - PUPPET_GEM_VERSION="~> 4.2.0"
    - PUPPET_GEM_VERSION="~> 4.3.0"
    - PUPPET_GEM_VERSION="~> 4.4.0"
</pre>

In the end .travis.yml should look close too https://github.com/Phil-Friderici/puppet-module-afs/blob/master/.travis.yml